index.html

```diff
@@ -27,69 +28,75 @@
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </div>
         <div class="text-content">
           <h4>About</h4>
-          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </div>
       </div>
-  
+
       <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
-  
+
       <div class="bottom-content">
         <div class="text-content">
           <h4>Services</h4>
-          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </div>
         <div class="text-content">
           <h4>Product</h4>
-          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </div>
         <div class="text-content">
           <h4>Vision</h4>
-          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </div>
       </div>
-  
+
     </section>
-  
+
     <section class="contact">
-  
+
       <h4>Contact</h4>
-  
+
       <p>123 Way 456 Street<br> Somewhere, USA</p>
       <br>
       <p>1 (888) 888-8888</p>
       <br>
       <p>sales@greatidea.io</p>
-  
+
     </section>
-  
+
     <footer>
       <p>Copyright Great Idea! 2018</p>
     </footer>
-  
+
   </div><!-- container -->
-    
-  
+
+
 </body>
+
```

Comment:
Good job changing the structure.  It is important to keep your code looking clean as possible.